bin/pycbc_optimal_snr

```diff
@@ -46,41 +46,53 @@ class TimeIndependentPSD(object):
 class TimeVaryingPSD(object):
     def __init__(self, file_name, length=None, delta_f=None, f_low=None):
         f = h5py.File(file_name, 'r')
+        self.file_name = file_name
         detector = f.keys()[0]
         self.start_times = f[detector + '/start_time'][:]
         self.end_times = f[detector + '/end_time'][:]
-        file_f_low = f.attrs['low_frequency_cutoff']
+        self.file_f_low = f.attrs['low_frequency_cutoff']
         f.close()
-        self.psds = []
-        for i in xrange(len(self.start_times)):
-            group = detector + '/psds/' + str(i)
-            psd = load_frequencyseries(file_name, group=group)
+        self._curr_psd = {}
+        self._curr_psd_index = {}
+        self.detector = detector
+        self.length = length
+        self.delta_f = delta_f
+        self.f_low = f_low
+
+    def __call__(self, time=None):
+        mask = np.logical_and(self.start_times <= time,
+                              self.end_times > time)
+        # Tito, what if this is bigger than 1?
```

Comment:
Then we have two overlapping PSDs, and I don't know what to do there... Raise an exception?